tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "moduleResolution": "node",
     "module": "commonjs",
     "noImplicitAny": false,
+    "esModuleInterop": true,
```

Comment:
this was required by: https://github.com/apollographql/apollo-server/issues/1189